### DISCUSSION TITLE
How do i resolve syntax error

### DISCUSSION THREAD

**Discussion by Bar-bara** [SERVIR-WA/GALUP#327] [Q&A] [answered]
How do I resolve this error after changing the image expression from NDVI to EVI?
![image](https://user-images.githubusercontent.com/85511497/144078028-e4d8b89a-103e-4136-9c4d-891c0a58d7a6.png)
![image](https://user-images.githubusercontent.com/85511497/144078403-de1ed967-ff0c-4ab0-b7bf-43a329174ac1.png)

**Comment by ecodynlab** [comment, marked as answer]
Hi Barbara, try deleting lines 72 through 74 and rerun the code. Let us know if this works.
Best,
Aditya
